bin/changelog

```diff
@@ -0,0 +1,301 @@
+#!/usr/bin/env node
+
+/* eslint-disable no-restricted-globals, no-console */
+
+const path = require('path');
+const { promises: fs } = require('fs');
+const util = require('util');
+
+const chalk = require('chalk');
+const execa = require('execa');
+const inquirer = require('inquirer');
+const lodash = require('lodash');
+const glob = util.promisify(require('glob'));
+const moment = require('moment');
+const yargs = require('yargs/yargs');
+const { hideBin } = require('yargs/helpers');
+
+const hotConfig = require('../hot.config');
+
+const isTTY = process.config.isTTY;
```

Comment:
Shouldn't be `process.stdout.isTTY`? https://nodejs.org/api/tty.html#tty_tty

Reply:
Good catch https://github.com/handsontable/handsontable/pull/7448/commits/377c44828ab6681cb8c4a694f830f3b0cfe16f17